terraform/gcp/modules/external_secrets/external_secrets.tf

```diff
@@ -76,14 +69,6 @@ spec:
   provider:
       gcpsm:
         projectID: "${var.project_id}"
-        auth:
```

Comment:
Is this deletion is intentional? (I don't understand this part of the change so this may be a dumb question.)

Reply:
this part is intentional, as we will be authenticating based on the service account attached to the external secrets deployment/pod instead of specifying the full authentication separately.

this is outlined here, https://external-secrets.io/v0.5.8/provider-google-secrets-manager/#using-pod-based-workload-identity

Reply:
Great, thanks for the explanation.